pytest.ini

```diff
@@ -1,4 +1,4 @@
 [pytest]
 pythonpath = .
-log_cli=true  
-log_level=INFO
\ No newline at end of file
+log_cli = true  
```

Comment:
Thought that "=" is better than " = "? A whatever, but made me thinking

Reply:
I like it more with spaces, it just looks better.
It depends, in some places it matters, in others it doesn't.
If I can choose I would go for a variant with spaces any day.